java/query/query-core/src/main/java/sleeper/query/core/tracker/QueryTrackerStore.java

```diff
@@ -18,14 +18,48 @@
 
 import java.util.List;
 
+/**
+ * Query tracker store.
```

Comment:
This is just the interface name. We can explain what this is for. It retrieves the status of queries.